Gem/Code/Source/EntityLatticeTestComponent.h

```diff
@@ -34,13 +36,20 @@ namespace AtomSampleViewer
         //! Returns total number of instances (width * height * depth)
         uint32_t GetInstanceCount() const;
         
+        //! Returns world space Aabb for the lattice.
```

Comment:
Maybe indicate the AABB does not encompass the entity models but only their origins.

Reply:
I prefer this to making the lattice AABB include the mesh AABB.

Reply:
Comment was updated.